### PR TITLE
mark polys 'current' if they match the active assignable property

### PR DESCRIPTION
**Pull request by bgschiller**
[demo video](https://trello-attachments.s3.amazonaws.com/58d428743111af1d0a20cf28/5a29b65a613d9d11d9d08ffb/202e782eed3f1e693a9f6ffed840ab99/recording.webm)

Addresses #261

### CODE REVIEW

File: src/components/Grid/Grid.vue [hunk 1]
```diff
@@ -377,6 +382,8 @@ export default {
               })),
             ...extraPolygonAttrs,
           };
+          console.log('model', model);
```
Comment (gh-23378) [inline src/components/Grid/Grid.vue]: Could we get rid of this?

Comment (bgschiller) [reply src/components/Grid/Grid.vue]: yeah...

**Comment by shorowit** [comment]
Cool! The outline can be difficult to see depending on the space color though. Would it stand out more if it was just black?

**Comment by macumber** [comment]
Looks good to me

**Comment by bgschiller** [comment]
@shorowit sure! I was reusing the same outline style as for the current space, so that's now black as well. I can change it back to red if needed.

**Pull request merged**